pages/linux/daemonize.md

```diff
@@ -0,0 +1,20 @@
+# daemonize
+
+> Run a command (that does not daemonize itself) as a Unix daemon.
+> More information: <http://software.clapper.org/daemonize/>.
+
+- Run command as a daemon:
+
+`daemonize {{command}} {{command_arguments}}`
+
+- Run command as a daemon, writing its pid to the specified file:
```

Comment:
Perhaps this will read better?
```suggestion
- Writing the pid to the specified file:
```

Also, is there a long form of this argument (`-p`) and the ones in the following examples? In tldr pages we prefer long-form options, as they are more descriptive.

Reply:
Unfortunately, `daemonize` doesn't have long-form alternate options, per its man page.